perfrepo-cli

```diff
@@ -347,6 +354,55 @@ class TestExecCLI(GenericCLI):
         self._perf_api.testExecution_update(testexec)
         return 0
 
+    def _do_search(self, argv):
+        i = 1
+
+        search_criteria = perfrepo.PerfRepoTestExecutionSearch()
+
+        # TODO: group, order-by, limit-from, how-many,
+        #       executed-before, executed-after
+
+        while i < len(argv):
+            if argv[i] == "ids":
+                ids = argv[i+1].split(',')
+                search_criteria.set_ids(ids)
+                i += 2
+            elif argv[i] == "uid":
+                search_criteria.set_testUid(argv[i+1])
+                i += 2
+            elif argv[i] == "name":
+                search_criteria.set_testName(argv[i+1])
+                i += 2
+            elif argv[i] == 'tags':
+                i += 1
+                while i < len(argv) and \
+                    argv[i] not in [ 'uid', 'name', 'param', 'id' ]:
```

Comment:
The indentation of this condition could be better... I don't like when a multi-line condition starts
on the same indentation level as the inner code block